cmd/geth/accountcmd.go

```diff
@@ -264,6 +264,9 @@ func accountProofOfPossession(ctx *cli.Context) error {
 	ks := am.Backends(keystore.KeyStoreType)[0].(*keystore.KeyStore)
 
 	signer := common.HexToAddress(ctx.Args()[0])
+	if !common.IsHexAddress(ctx.Args()[1]) {
```

Comment:
tiny nit: wdyt of putting `ctx.Args()[1]` into a variable so we only have to call `ctx.Args()[1]` twice? That way if the command args ever change in the future we don't have to change multiple places